cli/src/client/mod.rs

```diff
@@ -0,0 +1,44 @@
+use graphql_client::{GraphQLQuery, Response};
+use http::Uri;
+use reqwest::blocking;
+use std::error::Error;
+
+mod queries;
+
+static PROD_GQL_API_URL: &str = "https://engine-graphql.apollographql.com/api/graphql";
```

Comment:
I tend to think that urls should be backed by a possible env variable override both for test suites and for testing against our own stacks. What do you think?

Reply:
Yup, I can do that, and default to this value. It just means using a function because you can't have a static string get a value from the environment.

Reply:
Do we want to bake this into the config? We could then piggy back of env_var overrides we use there, and we would have a consistent environment variable naming scheme.

Reply:
@jsegaran I don't follow what you mean. What is the naming scheme? See the update to the PR and lmk if that works for you?